.github/ISSUE_TEMPLATE/bugs-performance-issues.md

```diff
@@ -0,0 +1,42 @@
+---
+name: Bug report (including performance and build issues)
+about: Help us improve the GPflow code base
+labels: bug
+---
+
+<!-- Lines like this are comments and will be invisible -->
+
+# Bug / performance issue / build issue
+
+<!-- A clear and concise description of what the bug is. -->
+
+## To reproduce
+
+**Minimal, reproducible example**
+<!-- We need to be able to reproduce the bug by simply copy&pasting your code -->
+```python
+# This is the place for your code that reproduces the bug.
+# Please make sure it does not depend on external libraries, specific datasets, etc., and the smaller, the better :)
+# For more information on good bug reports, check https://stackoverflow.com/help/minimal-reproducible-example
+```
+
+**Stack trace/error message**
+```
+// Paste the full stack trace/error message here
+```
+
+## Expected behavior
+
+<!-- A clear and concise description of what you expected to happen. -->
+
+## System information:
+
+* GPflow version: <!-- run `import gpflow; print(gpflow.__version__)` inside python -->
+* GPflow installed from: <!-- e.g. with "pip install gpflow" or from a GitHub clone on the develop branch; if working from a git clone, please include the git commit (SHA hash): run `git rev-parse HEAD` on the command line -->
```

Comment:
does the user need to specify the hash if they're using a tag?

Reply:
I guess tag would be fine, but tags are easily confused with branches, which keep moving